static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-design-variations.js

```diff
@@ -0,0 +1,126 @@
+define([
+    'common/modules/commercial/contributions-utilities',
+    'lib/template',
+    'lodash/objects/assign',
+    'raw-loader!common/views/acquisitions-epic-design-variations.html'
+], function(
+    contributionUtilities,
+    template,
+    assign,
+    acquisitionEpicDesignVariations
+) {
+
+    // Building the Epic component using the template acquisitions-epic-design-variations.html
+    // =======================================================================================
+
+    // Default arguments for the template.
+    function defaultTemplateArguments() {
```

Comment:
As this just returns an `Object` and doesn't take any arguments, you can remove the function definition. 
ie
```
var defaultTemplateArguments = {
     p1: '',
     p1Highlight: '',
     p1PostHighlight: '',
     ...
}
```